app/BUILD.gn

```diff
@@ -36,5 +50,70 @@ brave_grit("brave_generated_resources_grit") {
   }
 }
 
+source_set("browser_tests") {
+  testonly = true
+
+  if (!is_android && !is_ios) {
```

Comment:
Sorry, I realize this is an old PR, but I'm trying to understand this part. Is it intentional to only build this browser test on desktop? Unless I'm not understanding, this browser test is definitely meant to run on Android at least.

Reply:
Yeah, the code doesn't run on Android/iOS at all, so the test only runs on desktop too - it should probably have its own buildflag.

Reply:
OK, I think you're saying that `command_utils_browsertest` isn't supported on Android. But this change also prevents `brave_main_delegate_browsertest` from running on Android (even though it was previously supported there). Maybe I'm not understanding though?

Reply:
I have a PR that's addressing this, just trying to figure out if it was intentional. :-)

Reply:
That probably was not intentional :laughing:

Reply:
OK, no problem! I'm moving the feature tests into a unit test and also making them run on Android in this PR: https://github.com/brave/brave-core/pull/22516.